homeassistant/components/myfitnesspal/__init__.py

```diff
@@ -0,0 +1 @@
+"""The myfitnesspal component."""
```

Comment:
tests are needed

Reply:
Tests are underway but I can't seem to find a way to mock the client and get the sensor to update. Currently update is never run (I broke it intentionally and its unaffected)

Reply:
I don't have a ton of experience in python mocking but I did one a while back for praw here tests\components\reddit\test_sensor.py